www/public/site.css

```diff
@@ -367,7 +367,11 @@ h4 {
   font-family: 'Permanent Marker';
   font-style: normal;
   font-weight: 400;
-  src: url("fonts/PermanentMarker.woff2") format('woff2');
+  src: url('../fonts/permanent-marker-v16-latin/permanent-marker-v16-latin-regular.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
```

Comment:
There is no to support IE8 anymore right?

Reply:
if we want we can leave just woff + woff2 variants. should be enough for all modern browsers.

Reply:
Sounds good, browsers that don't support it will have their own fallback fonts so this should never be a real problem.